PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/Controllers/StockFiles/StocksViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  StocksViewController.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by 马文龙 on 9/5/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
+//
+
+import UIKit
+
+class StocksViewController: UIViewController {
+    var stocks = [StockModel]() {
+        didSet {
+            stockTableView.reloadData()
+        }
+    }
+    var sectionDate = [String]()
+    var monthString = ""
+    var yearString = ""
+    var monthNumberToWord = ""
+    var stocksArray = [[StockModel]]()
+    var monthOpenAmounts = [[Double]]()
+    var averages = [Double]()
+    
+    @IBOutlet weak var stockTableView: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        stockTableView.dataSource = self
+        loadData()
+    }
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard let indexPath = stockTableView.indexPathForSelectedRow,
+            let stocksDetailViewController = segue.destination as? StocksDetailViewController else {return}
+        let stock = stocksArray[indexPath.section][indexPath.row]
+        stocksDetailViewController.stockSelected = stock
+    }
+    func getDateMonth(dateString: String) {
```

Comment:
Great job handling the section names. Super impressive. However, consider refactoring and doing this logic in the model and not in the VC. It makes it a lot cleaner.